app/src/main/kotlin/at/bitfire/davdroid/ui/AccountsScreen.kt

```diff
@@ -293,6 +306,14 @@ fun AccountsScreen(
                                     val intent = Intent(Settings.ACTION_INTERNAL_STORAGE_SETTINGS)
                                     if (intent.resolveActivity(context.packageManager) != null)
                                         context.startActivity(intent)
+                                },
+                                calendarProviderDisabled = calendarProviderDisabled,
+                                onManageCalendarProvider = {
+                                    uriHandler.openUri(manualIntroductionUri.toString())
```

Comment:
Hm I wonder whether we should open _Android Settings / Apps_ (ideally even to the details of the correct app) instead of the manual? Then people could enable it right there.

Reply:
Would that work when the app is uninstalled?

Reply:
Guess we can do that if it's installed and point the users to the manual if it's not installed.